markdownlint/TOP003_defaultSectionContent/TOP003_defaultSectionContent.js

```diff
@@ -27,92 +28,153 @@ function getListSectionErrors(sectionTokens, section) {
+        deleteCount: tokensAfterHeading[0].line.length,
+        insertText: replacementText,
+      })
+    );
+  }
 
   const tokensAfterFirstContent = tokensAfterHeading.slice(
     tokensAfterHeading.findIndex(
       (token, _index, arr) =>
         token.type === arr[0].type.replace("_open", "_close")
     ) + 1
   );
+  const bulletListOpenTokenIndex = sectionTokens.findIndex(
+    (token) => token.type === "bullet_list_open"
+  );
   if (
-    hasDefaultContent &&
-    tokensAfterFirstContent[0].type !== "bullet_list_open"
+    (defaultContentOpenTokenIndex === 0 && !tokensAfterFirstContent.length) ||
+    bulletListOpenTokenIndex === -1
   ) {
+    const isAdditionalResources =
+      section === sectionsWithDefaultContent.additionalResources;
+    const tokenLineNumber = (
+      tokensAfterFirstContent[0] || tokensAfterHeading[0]
+    ).lineNumber;
+    const errorDetail = isAdditionalResources
+      ? `Expected section to include unordered list item: "It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum."`
+      : `Must include an unordered list of ${listItemsName} in the "${section}" section`;
     listSectionErrors.push(
       createErrorObject(
-        tokensAfterFirstContent[0].lineNumber,
-        `Only an unordered list of ${listItemsName} should follow the default content.`
+        tokenLineNumber,
+        errorDetail,
+        isAdditionalResources
+          ? {
+              lineNumber: !tokensAfterFirstContent.length
+                ? tokenLineNumber + 1
+                : tokenLineNumber,
+              insertText:
+                "\n- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.",
+            }
+          : {}
       )
     );
   }
 
-  const bulletListOpenTokenIndex = sectionTokens.findIndex(
-    (token) => token.type === "bullet_list_open"
-  );
-  if (bulletListOpenTokenIndex === -1) {
-    const errorDetail =
-      section === sectionsWithDefaultContent.additionalResources
-        ? `Expected section to include unordered list item: "It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum."`
-        : `Must include an unordered list of ${listItemsName} in the "${section}" section`;
+  if (
+    defaultContentOpenTokenIndex === 0 &&
+    tokensAfterFirstContent.length &&
+    !tokensAfterFirstContent[0].type.endsWith("_list_open")
+  ) {
     listSectionErrors.push(
       createErrorObject(
-        (tokensAfterFirstContent[0] || tokensAfterHeading[0]).lineNumber,
-        errorDetail
+        tokensAfterFirstContent[0].lineNumber,
+        `Only an unordered list of ${listItemsName} can follow the default content.`,
+        {
+          lineNumber: tokensAfterFirstContent[0].lineNumber,
+          deleteCount: -1,
```

Comment:
Could we define a `WHOLE_LINE` const for this and line 176? Think it'll make it easier to understand than the arbitrary `-1`
```suggestion
          deleteCount: WHOLE_LINE,
```